crates/re_log_types/src/arrow_msg.rs

```diff
@@ -131,19 +131,20 @@ mod tests {
 
     #[test]
     fn arrow_msg_roundtrip() {
-        let row = DataRow::from_cells2(
+        let mut row = DataRow::from_cells2(
             MsgId::random(),
             "world/rects",
             [build_frame_nr(0.into())],
             1,
             (build_some_point2d(1), build_some_rects(1)),
         );
+        row.compute_all_size_bytes();
 
         let table_in = row.into_table();
-        let msg_in: ArrowMsg = (&table_in).try_into().unwrap();
+        let msg_in = table_in.as_arrow_msg().unwrap();
         let buf = rmp_serde::to_vec(&msg_in).unwrap();
         let msg_out: ArrowMsg = rmp_serde::from_slice(&buf).unwrap();
-        let table_out: DataTable = (&msg_out).try_into().unwrap();
+        let table_out = DataTable::from_arrow_msg(&msg_out, true).unwrap();
```

Comment:
gain, the `true` here is unreadable. Either change it to an enum, or use this trick:

```suggestion
        let supercharged = true;
        let table_out = DataTable::from_arrow_msg(&msg_out, supercharged).unwrap();
```